components/gc-subway/section-end-en.html

```diff
@@ -1,58 +1,44 @@
 ---
```

Comment:
What is the purpose of this function?  The subway pattern is usually self contained and should not need to be escaped.  The example makes the pattern look broken.  could the content still be forced into the same column width? 
![image](https://user-images.githubusercontent.com/9367162/218217925-3f53de2f-fe12-4535-8b59-1137acf452f6.png)

Reply:
@Garneauma could you let me know some examples of what elements broke the plugin?  This is the first I'm hearing/knowing about it.   I do hope it does get patched because again, it looks like its broken here.  If maybe there is a class that can keep the width the same, without resorting to col-md-8/col-md-4?

Reply:
@Garneauma  Okay thanks, because I realized that if you have this "end" feature, but also want to add related information, it doesn't seem to work because semantically you would put the related information after the pagination links, but if they are outside of the subway design, they no longer "float right".

Reply:
@Christopher-O This is why it is specified in the documentation that those two features cannot be implemented together: https://github.com/wet-boew/GCWeb/pull/2058/files#diff-588c6c74983f3776844bf5576148e39badcf927214777b3873ef5388e76b4938R116-R117

Reply:
I just saw that in passing as I was reviewing the governance, and why I added a phrase in my governance notes that it's important to review the governance before proceeding to the examples first.  Hopefully that can be rectified before becoming stable.